cobalt/script/v8c/v8c_global_environment.cc

```diff
@@ -97,9 +97,6 @@ V8cGlobalEnvironment::V8cGlobalEnvironment(v8::Isolate* isolate)
   isolate_->SetData(kIsolateDataIndex, this);
   DCHECK(isolate_->GetData(kIsolateDataIndex) == this);
 
-  isolate_->SetAllowCodeGenerationFromStringsCallback(
-      AllowCodeGenerationFromStringsCallback);
```

Comment:
I think this gives us an event to handle eval() related web spec. See https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:cobalt/script/v8c/v8c_global_environment.cc;drc=7a6537c86df4096546b2fcec40078e431cd0b6d8;l=374

Reply:
I should have looked before asking, but I found in revision history:

> Use Isolate::SetModifyCodeGenerationFromStringsCallback instead.
> See http://crbug.com/v8/10096.